### PR TITLE
Reshuffled files (EIB and NCO request)

### PR DESCRIPTION
**Pull request by ilianagenkova**
- updated build.sh  (allows running build.sh with explicit path, from any space, NCO has this already)
- increased memory for global dump in ecf trigger per NNCO request.

- moved /release_notes/Release_Notes.txt to /docs/Release_Notes.txt and shortened contents
- updated top level CMakeLists.txt to not install obsolete /release_notes

- kept /triggers 'as is' (i.e. didn't move it)
- did not make /ecf/CMakeLists.txt b/c it's not something that gets installed (similarly to /triggers, no CMakeLists.txt there )

Note that /triggers and /ecf are only used by EMC and NCO, respectively, so they exist for awareness.

### CODE REVIEW

File: docs/Release_Notes.txt [hunk 1]
```diff
@@ -0,0 +1,497 @@
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+obsproc v1.1.0        -> released to NCO on Aug ??,2022
+                      -> implemented ?? ??,2022 (prior or in sync with GFSv16.3)
+
+
+List of all code/scripts modified with this release
+---------------------------------------------------
+M CMakeLists.txt
+R build-obsproc/*
```
Comment (aerorahul) [inline docs/Release_Notes.txt]: I don't know what `R` is, but this directory does not exists any more.

File: docs/Release_Notes.txt [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+A cmake/FindMKL.cmake
+A cmake/FindNetCDF.cmake
+M docs/README.md
+  fix/CMakeLists.txt
+  fix/bufrtab.mods
+  fix/prepobs_anow.bufrtable
+  jobs/CMakeLists.txt
```
Comment (aerorahul) [inline docs/Release_Notes.txt]: Not sure what the purpose of listing every file in the release notes is. 
If anyone is interested in knowing which files have changed, a `git diff` can be performed.

Release notes should contain significant changes/upgrades in the product, bugfixes and any relevant information that the user needs to know e.g. API changes. 

What these jobs/scripts do belong in documentation, not release notes.

File: docs/Release_Notes.txt [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+  versions/build.ver
+M versions/run.ver
+M versions/VERSION
+------------------
+M-modified; A-added; D-deleted
+
+JOBS file changes
+-----------------
+ jobs/JOBSPROC_AIRNOW_DUMP
+ Correct COMIN path
```
Comment (aerorahul) [inline docs/Release_Notes.txt:45]: This (and similar information below) belongs in a ChangeLog, not release note.

File: docs/Release_Notes.txt [hunk 4]
```diff
@@ -0,0 +1,497 @@
...
+Disk Space Changes 
+------------------
+ The newly generated dumps ( *gmi1cr*, *snocvr*, *saldrn*, *subpfl* ) vary in size from 1Mb to 200Mb
+ depending on network, cycle time, time dump window. 0 size dumps are possible
+ The updated dumps (*gpsro*, *satwnd* and *nsstbufr) are up to ~25% larger in size
+ For examples, see: /lfs/h2/emc/stmp/iliana.genkova/CRON/RELv1.1.0/com
+ 
+Shared Software 
+---------------
+ obsproc needs modules bufr-dump and prepobs in order to run
```
Comment (aerorahul) [inline docs/Release_Notes.txt]: This is documentation, not Release Note material.

File: docs/Release_Notes.txt [hunk 5]
```diff
@@ -0,0 +1,497 @@
...
+obsproc v1.0.0        -> released to NCO on Dec 21,2021
+        v1.0.2        -> implemented Jun 28, 2022             
+
+Transition from WCOSS Cray and Phase3 to WCOSS2
+======================================
+https://docs.google.com/document/d/1U4pu4iSlLOnbNrcBqBlOU7bnNcTFfe_MpjV8uKN4S40/edit
+
+Where is the release tag on subversion/git/vlab? 
+ git clone https://github.com/NOAA-EMC/obsproc
+ git checkout release/obsproc.v1.0.0
```
Comment (aerorahul) [inline docs/Release_Notes.txt:120]: The fact that the user has already checked out this tag, means they already know this.
Also not release note material.

File: docs/Release_Notes.txt [hunk 6]
```diff
@@ -0,0 +1,497 @@
...
+ bacio_ver=2.4.1
+ bufr_dump_ver=1.0.0
+ gfs_ver=v16.2.0
+ omb_ver=v4.4.0
+ dictionaries_ver=v3.4.0
+
+List of all code/scripts modified with this release:
+A CMakeLists.txt
+A VERSION
+A build.sh
```
Comment (aerorahul) [inline docs/Release_Notes.txt:147]: This is not correct.  Also, again, why are all files changed needed to be listed in a Release Note?

File: docs/Release_Notes.txt [hunk 7]
```diff
@@ -0,0 +1,497 @@
...
+ w3nco_ver=2.4.1
+ bacio_ver=2.4.1
+ bufr_dump_ver=1.0.0
+ gfs_ver=v16.2.0
+ omb_ver=v4.4.0
+ dictionaries_ver=v3.4.0
+
+List of all code/scripts modified with this release:
+A CMakeLists.txt
+A VERSION
```
Comment (aerorahul) [inline docs/Release_Notes.txt:146]: Incorrect.  This was moved.

File: docs/Release_Notes.txt [hunk 8]
```diff
@@ -0,0 +1,497 @@
...
+
+Dissemination will remain the same unless otherwise communicated to NCO and a PNS/SCN issued
+
+HPSS archive retention will remain the same unless approval granted by HPCRAC
+
+What are your up and downstream dependencies?
+ Upstream: decoders, satingest ($DCOMROOT)
+ Downstream: gfs and gdas model and analysis runs; omb sice data
+
+Provide a pointer to your COMOUT, temp directory and pbspro output file that was used during testing:
```
Comment (aerorahul) [inline docs/Release_Notes.txt:328]: I am not sure I understand the point of this in a Release Note.

**Review by aerorahul — CHANGES_REQUESTED** [review]
A few comments:
- Information in the release notes is incorrect or dated and that should be corrected (and that is the reason for requesting changes)
- On a larger point, Release notes is not the place for a lot of the information that has been placed in this file.  The release notes reads like a ChangeLog. 
- There are items in the Release Notes that clearly belong in Documentation. 

Re. the omission of `CMakeLists.txt` in `ecf` with the reason being EMC or NCO don't install them.
That is true today, however, in that case, one can argue removing `CMakeLists.txt` from everywhere except `sorc`, since we don't install `ush/`, `jobs/` or `scripts/` or `fix/` either.

File: docs/Release_Notes.txt (hunk 1 above)
Comment (ilianagenkova) [reply docs/Release_Notes.txt]: R(removed) replaced with D (deleted)

File: docs/Release_Notes.txt (hunk 2 above)
Comment (ilianagenkova) [reply docs/Release_Notes.txt]: will shorten the list of files to the ones with changes only

File: docs/Release_Notes.txt (hunk 3 above)
Comment (ilianagenkova) [reply docs/Release_Notes.txt:45]: respectfully, I find this comment contradicting your previous comment on what RN should contain

File: docs/Release_Notes.txt (hunk 4 above)
Comment (ilianagenkova) [reply docs/Release_Notes.txt]: it's in the /doc folder. further consideration to your comments will be given when time allows

File: docs/Release_Notes.txt (hunk 5 above)
Comment (ilianagenkova) [reply docs/Release_Notes.txt:120]: This is an older part of RN, which was already released, so I'll keep it for the records

File: docs/Release_Notes.txt (hunk 8 above)
Comment (ilianagenkova) [reply docs/Release_Notes.txt:328]: Reply to this and the earlier comment - these were the questions posed by NCO, we answered them. I can't answer for past decisions before my time.

**Comment by ilianagenkova** [comment]
@aerorahul , appreciate your input, yet we will make only some changes due to time constrains and the code already  running on para. thanks! Also, like the markdown idea, but will use for RN in next release, so all obsproc suite codes start using markdown at the same tme.

File: docs/Release_Notes.txt (hunk 7 above)
Comment (ilianagenkova) [reply docs/Release_Notes.txt:146]: This part of the RN (after line 110) are the RN from the transition to WCOSS2. We keep all RNs in one file for easier & faster search.

**Comment by ilianagenkova** [comment]
Updated /ecf with NCO's para /lfs/h1/ops/para/packages/obsproc.v1.1.0/ecf
Uncommented DBnet for global dump

**Pull request merged**